pkg/render/fluentd_test.go

```diff
@@ -241,12 +250,12 @@ var _ = Describe("Tigera Secure Fluentd rendering tests", func() {
 		}
 		instance.Spec.AdditionalStores = &operatorv1.AdditionalLogStoreSpec{
 			Splunk: &operatorv1.SplunkStoreSpec{
-				Endpoint:    "https://1.2.3.4:8088",
+				Endpoint: "https://1.2.3.4:8088",
 			},
 		}
 		component := render.Fluentd(instance, nil, esConfigMap, s3Creds, splkCreds, filters, eksConfig, nil, installation)
 		resources, _ := component.Objects()
-		Expect(len(resources)).To(Equal(4))
+		Expect(len(resources)).To(Equal(7))
```

Comment:
Ok I'm going to stop saying Same here. Please fix up the rest of the Expect(len(resources)) and the resources[X] places. Thanks.